CHANGELOG.md

```diff
@@ -1,82 +1,142 @@
-- Added lowercase of åäö in entityMap.
-- Moved existing sources into `lib` subdirectory.
-- Removed `.npmignore` in favor of `files` entry in package.json.
-- More `package.json` refactoring.
-- Replaced CHANGELOG with more rigorous file.
-- Replaced LICENSE with more rigorous file.
-- Removed component.json (deprecated package manager https://github.com/componentjs/guide)
-- `proof` devDep updated to latest.
-- Fixed CI.
+[Commits](https://github.com/xmldom/xmldom/compare/0.2.1...0.3.0)
+
+- **BREAKING** Node >=10.x is now required.
+- **BREAKING** Remove `component.json` (deprecated package manager https://github.com/componentjs/guide)
+- **BREAKING** Move existing sources into `lib` subdirectory.
+- **POSSIBLY BREAKING** Introduce `files` entry in `package.json` and remove use of `.npmignore`.
+- [Add `Document.getElementsByClassName`](https://github.com/xmldom/xmldom/issues/24).
+- [Add `Node` to the list of exports](https://github.com/xmldom/xmldom/pull/27)
+- [Add lowercase of åäö in `entityMap`](https://github.com/xmldom/xmldom/pull/23).
+- More `package.json` cleanup/refactoring.
+- Move CHANGELOG to markdown file.
+- Move LICENSE to markdown file.
+- Fix travis CI.
 - README updates.
 
 ## 0.2.1
 
-- More package.json refactoring.
+[Commits](https://github.com/xmldom/xmldom/compare/0.2.0...0.2.1)
+
+- Correct `homepage`, `repository` and `bugs` URLs in `package.json`.
 
 ## 0.2.0
 
-- Now publishing under xmldom npm package again; retiring xmldom-alpha.
+[Commits](https://github.com/xmldom/xmldom/compare/v0.1.27...0.2.0)
+
+**Includes all BREAKING changes introduced in [`xmldom-alpha@v0.1.28`](#0128) and in later commits to master by the original authors.**
+
+- **POSSIBLY BREAKING** [removes the `Object.create` check from the `_extends` method of `dom.js` that added a `__proto__` property](https://github.com/xmldom/xmldom/commit/0be2ae910a8a22c9ec2cac042e04de4c04317d2a#diff-7d1c5d97786fdf9af5446a241d0b6d56L19-L22) ().
+- **POSSIBLY BREAKING** [removes code that added a `__proto__` property](https://github.com/xmldom/xmldom/commit/366159a76a181ce9a0d83f5dc48205686cfaf9cc)
+- formatting/corrections in `package.json`
 - Remove coveralls.
 - Disable cache in travis.
 - Other refactoring.
 
-## 0.1.28 (via xmldom-alpha npm package)
+## 0.1.31
+
+[Commits](https://github.com/xmldom/xmldom/compare/v0.1.27...v0.1.31)
+
+The patch versions (`v0.1.29` - `v0.1.31`) that have been released on the [v0.1.x branch](https://github.com/xmldom/xmldom/tree/0.1.x), to reflect the changed maintainers, **are branched off from [`v0.1.27`](#0127) so they don't include the breaking changes introduced in [`xmldom-alpha@v0.1.28`](#0128)**:
+
+## Maintainer changes
+
+After the last commit to the original repository <https://github.com/jindw/xmldom> on the 9th of May 2017, the first commit to this fork is from the 19th of December 2019. [The fork has been announced in the original repository on the 2nd of March 2020.](https://github.com/jindw/xmldom/issues/259)
```

Comment:
Subjective: I am torn at this point whether to refer to this repo as a "fork" or as a new project location.

Assuming we continue to publish into the same `xmldom` package on npm, I would not really favor calling this a "fork".

OTOH if we would start publishing this as multiple packages in the `@xmldom` namespace on npm, which I had already reserved as an org (see issue #54), then I think "fork" would be appropriate.

Reply:
what about replacing `this fork` with`<https://github.com/xmldom/xmldom>` so it stays correct even if it should ever move somewhere else.

I think "the announcement of the fork" is still the correct wording for describing what happened back then.